src/providers/seventv/SeventvEmotes.cpp

```diff
@@ -131,114 +139,44 @@ boost::optional<EmotePtr> SeventvEmotes::emote(const EmoteName &name) const
 
 void SeventvEmotes::loadEmotes()
 {
-    qCDebug(chatterinoSeventv) << "Loading 7TV Emotes";
-
-    QJsonObject payload, variables;
-
-    QString query = R"(
-        query loadGlobalEmotes($query: String!, $globalState: String, $page: Int, $limit: Int, $pageSize: Int) {
-        search_emotes(query: $query, globalState: $globalState, page: $page, limit: $limit, pageSize: $pageSize) {
-            id
-            name
-            provider
-            provider_id
-            visibility
-            mime
-            owner {
-                id
-                display_name
-                login
-                twitch_id
-            }
-        }
-    })";
-
-    variables.insert("query", QString());
-    variables.insert("globalState", "only");
-    variables.insert("page", 1);  // TODO(zneix): Add support for pagination
-    variables.insert("limit", maxPageSize);
-    variables.insert("pageSize", maxPageSize);
+    qCDebug(chatterinoSeventv) << "[7TVEmotes] Loading 7TV Global Emotes";
 
-    payload.insert("query", query.replace(whitespaceRegex, " "));
-    payload.insert("variables", variables);
+    QString url("https://api.7tv.app/v2/emotes/global");
```

Comment:
Would suggest moving this URL directly into the `NetworkRequest(...)` call instead of creating a variable as it's never used later on. (GH doesn't allow suggestions across deleted lines, sorry)

Reply:
Sure, I based this off the BTTV module. I will change there too.